### PR TITLE
chore(compaction): put all compaction metrics into a grafana graph

### PR DESCRIPTION
**Pull request by soundOfDestiny**
## What's changed and what's your intention?

- put all compaction metrics into a grafana graph

### CODE REVIEW

File: grafana/risingwave-dashboard.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
 	"fiscalYearStartMonth": 0,
 	"graphTooltip": 0,
 	"id": 2,
-	"iteration": 1648803156530,
+	"iteration": 1650019185521,
```
Comment (hzxa21) [inline grafana/risingwave-dashboard.json:62]: Just curious, what is the meaning of `iteration`?

**Comment by hzxa21** [comment]
Cool! We now have IO metrics for compaction!

image

@soundOfDestiny However, when I tried running e2e using your branch, it looks like some of the metrics always remain zero even after compaction ran successfully. Can you take a look?

**Comment by soundOfDestiny** [comment]
> Cool! We now have IO metrics for compaction!
> 
> image
> 
> @soundOfDestiny However, when I tried running e2e using your branch, it looks like some of the metrics always remain zero even after compaction ran successfully. Can you take a look?

stat logic of these metrics are temporarily removed in https://github.com/singularity-data/risingwave/pull/1328, and will be added back in a following PR

File: grafana/risingwave-dashboard.json (hunk 1 above)
Comment (wcy-fdu) [reply grafana/risingwave-dashboard.json:62]: That is randomly generated by grafana dashboard, every time someone changes dashboard and export it,` iteration` will be redistributed.

**Review by hzxa21 — APPROVED** [review]

**Pull request merged**